pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.testobject</groupId>
 	<artifactId>testobject-java-api</artifactId>
-	<version>0.0.26</version>
+	<version>0.0.27</version>
```

Comment:
Since we are breaking the api, we should increase the minor version. So maybe 0.1.0 makes more sense